Jessaleem/airbnb_backend
# [Feat]: Add controlers and routes, to store files on cloudinary.

### Description
[//]: <> (Aca debe ir la descripcion del PullRequest, que es? que hace?)

Added multer npm library , cloudinary library, and add route to upload images. Also inside of upload route added index that  specify routes to save images on front, and the controllers file. 


### Feeling
[//]: <> (Como te sientes con este PR? la solucion que entregas como te hace sentir?)
- [x] 🤞 I hope this works, please check it carefully. 

### Tickets
[//]: <> (El numero del ticket asociado a este PR)

### How to test ? 
[//]: <> (Pasos necesarios para probar esta funcionalidad)
run it in the terminal at the same time that the front is runnning, upload some photos, and after that check in cloudinary websites if your images hase been uploaded. 

### Screenshots 
[//]: <> (Capturas de pantalla que ayuden a entender que hiciste en este PR)

![image](https://user-images.githubusercontent.com/105081872/187252606-c627129a-3322-4ff5-a270-2cfdaacaf245.png)

![image](https://user-images.githubusercontent.com/105081872/187252703-d1a38a5c-8246-4ecd-9c8a-a638dc91d92c.png)


### Scope

- [x] ⚡️ New feature, no breaking change that adds a new functionality.
